wordpress-mobile/WordPress-FluxC-Android
# Fix duplicated Top Performers products

Summary
==========
Fix for the https://github.com/woocommerce/woocommerce-android/issues/3460. Sometimes the Top Performers view at the `My Store` section shows the same Top Performer product twice.

How to Test
==========
It still not clear when the site returns duplicated product information from the Top Performers endpoints, so there's no reliable way to reproduce that, so this step by step is just a suggestion about how normally this issue have been reproduced
Steps to reproduce the behavior:
1. Go to the Woo app at the `My Store` section and make sure the `Today` tab is selected (preferably make sure that no order was created)
2. Go to the Woo site and make a single order with a single product
3. Go back to the Woo app and do a scroll refresh
4. Verify if the same product shows up twice

Screenshot
==========
![screenshot-1610706461726](https://user-images.githubusercontent.com/5920403/105244404-3d10fe80-5b4f-11eb-8cd2-91b3c5bb50d7.jpg)


Update release notes:
